pandas/tests/tools/test_to_datetime.py

```diff
@@ -629,8 +629,8 @@ def test_to_datetime_iso_week_year_format(self, s, _format, dt):
         "msg, s, _format",
         [
             [
-                "ISO week directive '%V' must be used with the ISO year directive "
-                "'%G' and a weekday directive '%A', '%a', '%w', or '%u'.",
+                "ISO week directive '%V' is incompatible with the year directive "
+                "'%Y'. Use the ISO year '%G' instead.",
```

Comment:
I've put the "%Y and %V used together" check before the "%V used without both %G and weekday directive", as I think it produces a more useful error message in this case